### ISSUE TITLE
Need \usepackage{hyperref}

### ISSUE THREAD

**Issue by meresmclr** [matthieu-vergne/LaTeX#2]
\usepackage{url} doesn't work,  \usepackage{hyperref} is needed in your main .tex document
people might not realize they need to do this if they're already using \usepackage{url}

**Comment by matthieu-vergne** [comment]
True. It is a known issue and I would like to remove the dependency (or at least not require the user to add it himself) without loosing the links but I did not spend time looking for solutions. Any help on that matter would be appreciated.

**Comment by meresmclr** [comment]
I would say maybe just put that in the Readme to let people know at least?

**Comment by matthieu-vergne** [comment]
I will push that later, but this is not what I would call a solution. I only need links, so I assume we can do it by ourselves without this package.
